runs/play.py

```diff
@@ -109,7 +109,7 @@ def create_trial_params(trial_idx):
                                 actor_config_template=actor_params.get("agent_config", None),
                                 run_id=run_session.run_id,
                                 environment_specs=self._environment_specs.serialize(),
-                                seed=(self._cfg.seed + actor_idx) * trial_idx,
+                                seed=(self._cfg.seed + actor_idx) * trial_idx + actor_idx,
```

Comment:
Needs this in order for two random actors in the same game to have different actions. I think we should simply attribute specific seeds to each random actor in their configs but it wasn't done in this PR.

Reply:
yep it makes sense

Reply:
wait no it doesn't the actor_idx is already in the mix

Reply:
Sorry for the bad explanation, this is for the edge case of trial_idx=0 (the first trial).

Reply:
ok then may the fix should be to use something like `self._cfg.seed * (trial_idx + 1) * (actor_idx + 1)` as it is right now the seed is ignored for the first trial